### DISCUSSION TITLE
Specify config file location from command line

### DISCUSSION THREAD

**Discussion by bxyoung** [jagrosh/MusicBot#702] [Questions] [answered]
Is it possible to pass in the configuration file location from the command line?

**Example:**
java -Dnogui=true -jar C:\MusicBots\JMusicBot-0.3.4.jar -config C:\MusicBots\config01.txt

**Comment by jagrosh** [comment, marked as answer]
Yes, this is described in detail [on the wiki](https://github.com/jagrosh/MusicBot/wiki/Advanced-Config-Options#specify-a-different-config-file).

**Reply by bxyoung** [reply]
Thank you so much didn't see the wiki!
